src/nomad_simulations/schema_packages/properties/energies.py

```diff
@@ -20,33 +20,65 @@
 
 import numpy as np
 
-from nomad.metainfo import Quantity
+from nomad.metainfo import Quantity, Section, Context, SubSection
 
 if TYPE_CHECKING:
     from nomad.metainfo import Section, Context
     from nomad.datamodel.datamodel import EntryArchive
     from structlog.stdlib import BoundLogger
 
-from nomad_simulations.schema_packages.physical_property import PhysicalProperty
+from nomad_simulations.schema_packages.physical_property import (
+    PhysicalProperty,
+    PropertyContribution,
+)
 
+##################
+# Abstract classes
+##################
 
-class FermiLevel(PhysicalProperty):
+
+class BaseEnergy(PhysicalProperty):
     """
-    Energy required to add or extract a charge from a material at zero temperature. It can be also defined as the chemical potential at zero temperature.
+    Abstract physical property section describing some energy of a (sub)system.
```

Comment:
I think you can improve this description a bit. What about:

Abstract class used to define the common value of the different type of energies. This class is useful to avoid repeating the definitions of `value` for all the energies.